source/Octopus.Tentacle.Client/Execution/RpcCallExecutor.cs

```diff
@@ -89,28 +90,67 @@ internal RpcCallExecutor(RpcCallRetryHandler rpcCallRetryHandler, ITentacleClien
             Func<CancellationToken, T> action,
             ClientOperationMetricsBuilder clientOperationMetricsBuilder,
             CancellationToken cancellationToken)
+        {
+            return Execute(rpcCall, action, abandonActionOnCancellation: false, clientOperationMetricsBuilder, cancellationToken);
```

Comment:
Can we keep the interface the same as ExecuteWithRetries so the caller has to explicity decide if abandonActionOnCancellation is true or false and not get a default this overload provides